OpenOversight/app/models/config.py

```diff
@@ -100,6 +100,11 @@ def __init__(self):
         self.NUM_OFFICERS = 120
         self.RATELIMIT_ENABLED = False
         self.SQLALCHEMY_DATABASE_URI = "sqlite:///:memory:"
+        # Disable sqlite cached statements
+        # https://github.com/python/cpython/issues/118172
+        self.SQLALCHEMY_ENGINE_OPTIONS = {
+            "connect_args": {"cached_statements": 0},
+        }
```

Comment:
This change was required to fix our tests in https://github.com/OrcaCollective/OpenOversight/pull/533

It's included here since I carried the entire change over and it's a relatively small change. Happy to split it into a separate pull request if needed